javascript/big-practice/src/scripts/models/noteModel.js

```diff
@@ -31,34 +20,20 @@ export default class NoteModel {
    * @returns {Object} note
    */
   async addNote(title, description) {
-    let note = {};
-    const userId = this.localStorage.getItems(STORAGE_KEYS.USER_ID);
-    const patternNote = {
-      id: new Date().getTime().toString(),
-      title,
-      description,
-      deleteAt: '',
-    };
-
-    if (userId) {
+    try {
       const noteItem = {
-        ...patternNote,
-        ownerId: userId,
+        id: new Date().getTime().toString(),
+        title,
+        description,
+        deleteAt: '',
       };
-      note = noteItem;
 
       await postData(noteItem);
-    } else {
-      const noteItem = {
-        ...patternNote,
-      };
-      note = noteItem;
-
-      this.notes.push(note);
-      this.localStorage.setItems(STORAGE_KEYS.LIST_NOTE, this.notes);
+      return noteItem;
```

Comment:
Does `await postData(noteItem);` return any result? If yes, does it return the added `noteItem` data?